modules/openapi-generator/src/main/resources/typescript-fetch/modelEnum.mustache

```diff
@@ -1,12 +1,8 @@
-/**
- * {{{description}}}
- * @export
- * @enum {string}
- */
+{{#description}}// {{description}}{{/description}}
 export enum {{classname}} {
 {{#allowableValues}}
 {{#enumVars}}
-    {{{name}}} = <any> {{{value}}}{{^-last}},{{/-last}}
+    {{{name}}} = {{{value}}}{{^-last}},{{/-last}} {{#description}}// {{description}}{{/description}}
```

Comment:
Is there a reason why `<any>` was removed? 

This was not removed in `modelGeneric.mustache` (see below)

Reply:
No I think I must have just sweept it away when changeing things. I can't see a reason for it though, enum values should be true to what they really are without type erasure.